consent/manager_test_helpers.go

```diff
@@ -126,6 +126,40 @@ func MockLogoutRequest(key string, withClient bool, network string) (c *flow.Log
 	}
 }
 
+func MockDeviceRequest(key string, authAt bool, network string) (c *flow.DeviceUserAuthRequest, h *flow.HandledDeviceUserAuthRequest, f *flow.DeviceFlow) {
```

Comment:
nitpick(non-blocking): `authAt` parameter is not used in the function, so tests only rely on the numeric string value

Reply:
you are right